### ISSUE TITLE
Automatically produce forecasts for test set at training step

### ISSUE THREAD

**Issue by CarolineRHolmes** [icenet-ai/icenet#351]
Introduce option to automatically produce forecast for test (and optionally validation set) at the end of model training, by using the pre-existing configuration test dates. e.g. run_train_ensemble step of pipeline would call a prediction step after successful training.

**Comment by JimCircadian** [comment]
Thanks @CarolineRHolmes, this is a great idea. The main requirement is to add CLI arguments to ensure that the outputs are captured when the validation or test sets (or both, which doesn't currently happen) are run through `evaluate_model` and output to files. With these individual output as per a regular prediction, a NetCDF can be created normally thereafter.

Nice one! 👍🏼
